rosdep/base.yaml

```
@@ -5722,6 +5722,14 @@ libsensors4-dev:
 libserial-dev:
   debian: [libserial-dev]
   ubuntu: [libserial-dev]
+libshaderc-dev:
+  arch: [shaderc]
+  debian: [libshaderc-dev]
+  fedora: [libshaderc-devel]
+  gentoo: [media-libs/shaderc]
+  ubuntu:
+    '*': null
+    noble: [libshaderc-dev]
```

```suggestion
  ubuntu:
    '*': [libshaderc-dev]
    focal: null
    jammy: null
```

(so we don't have to update it for future ubuntu distributions)